### ISSUE TITLE
flutter packages get failed on path_provider dependency

### ISSUE THREAD

**Issue by leagmain** [flutter/flutter#28497]
## Steps
add 'path_provider' as dependency
```
dependencies:
  flutter:
    sdk: flutter
  path_provider:
```

## flutter run --verbose
```
[  +29 ms] executing: [/Users/leagmain/Applications/flutter/] git rev-parse --abbrev-ref --symbolic @{u}
[  +36 ms] Exit code 0 from: git rev-parse --abbrev-ref --symbolic @{u}
[        ] origin/stable
[        ] executing: [/Users/leagmain/Applications/flutter/] git rev-parse --abbrev-ref HEAD
[  +11 ms] Exit code 0 from: git rev-parse --abbrev-ref HEAD
[        ] stable
[        ] executing: [/Users/leagmain/Applications/flutter/] git ls-remote --get-url origin
[  +13 ms] Exit code 0 from: git ls-remote --get-url origin
[        ] git@github.com:flutter/flutter.git
[   +1 ms] executing: [/Users/leagmain/Applications/flutter/] git log -n 1 --pretty=format:%H
[  +16 ms] Exit code 0 from: git log -n 1 --pretty=format:%H
[        ] 5391447fae6209bb21a89e6a5a6583cac1af9b4b
[        ] executing: [/Users/leagmain/Applications/flutter/] git log -n 1 --pretty=format:%ar
[  +22 ms] Exit code 0 from: git log -n 1 --pretty=format:%ar
[        ] 3 months ago
[   +1 ms] executing: [/Users/leagmain/Applications/flutter/] git describe --match v*.*.* --first-parent --long --tags
[  +14 ms] Exit code 0 from: git describe --match v*.*.* --first-parent --long --tags
[        ] v1.0.0-0-g5391447fa
[ +345 ms] executing: /Users/leagmain/Library/Android/sdk/platform-tools/adb devices -l
[  +13 ms] Exit code 0 from: /Users/leagmain/Library/Android/sdk/platform-tools/adb devices -l
[        ] List of devices attached
           emulator-5554          device product:sdk_gphone_x86 model:Android_SDK_built_for_x86 device:generic_x86 transport_id:1
[  +11 ms] executing: idevice_id -h
[   +3 ms] /usr/bin/xcrun simctl list --json devices
[ +306 ms] Running "flutter packages get" in demo...
[   +3 ms] executing: [/Users/leagmain/workspace/demo/] /Users/leagmain/Applications/flutter/bin/cache/dart-sdk/bin/pub --verbosity=warning --verbose get --no-precompile
[+3878 ms] HTTP error 424: Failed Dependency
[  +14 ms] package:pub/src/http.dart 218:5                 _ThrowingClient.send
[        ] ===== asynchronous gap ===========================
[        ] dart:async                                      _AsyncAwaitCompleter.completeError
[        ] package:pub/src/http.dart                       _ThrowingClient.send
[        ] ===== asynchronous gap ===========================
[        ] dart:async                                      _asyncThenWrapperHelper
[        ] package:pub/src/http.dart                       _ThrowingClient.send
[        ] package:http_throttle/http_throttle.dart 33:31  ThrottleClient.send
[        ] ===== asynchronous gap ===========================
[        ] dart:async                                      _asyncThenWrapperHelper
[        ] package:http_throttle/http_throttle.dart        ThrottleClient.send
[        ] package:pub/src/source/hosted.dart 301:37       BoundHostedSource._download
[        ] ===== asynchronous gap ===========================
[        ] dart:async                                      _asyncThenWrapperHelper
[        ] package:pub/src/entrypoint.dart                 Entrypoint.acquireDependencies
[        ] package:pub/src/command/get.dart 41:23          GetCommand.run
[        ] package:args/command_runner.dart 194:27         CommandRunner.runCommand
[        ] This is an unexpected error. Please run
[        ]     pub --trace --verbosity=warning --verbose get --no-precompile
[        ] and include the logs in an issue on https://github.com/dart-lang/pub/issues/new
[  +11 ms] pub get failed (69) -- attempting retry 1 in 1 second...
```

## flutter analyze

```
Running "flutter packages get" in demo...                        
HTTP error 424: Failed Dependency
package:pub/src/http.dart 218:5                 _ThrowingClient.send
===== asynchronous gap ===========================
package:http_throttle/http_throttle.dart 33:31  ThrottleClient.send
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 301:37       BoundHostedSource._download
===== asynchronous gap ===========================
package:pub/src/source/hosted.dart 196:13       BoundHostedSource.downloadToSystemCache
===== asynchronous gap ===========================
package:pub/src/entrypoint.dart 371:48          Entrypoint._get.<fn>
===== asynchronous gap ===========================
dart:async                                      runZoned
package:pub/src/http.dart 272:10                withDependencyType
package:pub/src/entrypoint.dart 367:12          Entrypoint._get
dart:async                                      Future.wait
package:pub/src/entrypoint.dart 228:18          Entrypoint.acquireDependencies
dart:async                                      _completeOnAsyncReturn
package:pub/src/solver/version_solver.dart      VersionSolver.solve
dart:async                                      _completeOnAsyncReturn
package:pub/src/solver/version_solver.dart      VersionSolver._result
This is an unexpected error. Please run

    pub --trace --verbosity=warning get --no-precompile

and include the logs in an issue on https://github.com/dart-lang/pub/issues/new
```
## flutter doctor -v
```
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.14.3 18D109, locale en-CN)
    • Flutter version 1.0.0 at /Users/leagmain/Applications/flutter
    • Framework revision 5391447fae (3 months ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[✓] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/leagmain/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    • All Android licenses accepted.

[!] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    ✗ libimobiledevice and ideviceinstaller are not installed. To install with Brew, run:
        brew update
        brew install --HEAD usbmuxd
        brew link usbmuxd
        brew install --HEAD libimobiledevice
        brew install ideviceinstaller
    ✗ ios-deploy not installed. To install with Brew:
        brew install ios-deploy
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without resolving iOS dependencies with CocoaPods, plugins will not work on iOS.
        For more info, see https://flutter.io/platform-plugins
      To install:
        brew install cocoapods
        pod setup

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 33.3.1
    • Dart plugin version 182.5215
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[!] IntelliJ IDEA Community Edition (version 2018.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • For information about installing plugins, see
      https://flutter.io/intellij-setup/#installing-the-plugins

[✓] VS Code (version 1.31.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.23.1

[✓] Connected device (1 available)
    • Android SDK built for x86 • emulator-5554 • android-x86 • Android 8.1.0 (API 27) (emulator)

! Doctor found issues in 2 categories.
```

**Comment by zoechi** [comment]
Looks like a duplicate of #28498
There are a lot of such reports lately.

**Comment by chenglu** [comment]
It has been fixed, see my comments on [#28498](https://github.com/flutter/flutter/issues/28498#issuecomment-467462967) for more information.
